release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -548,11 +548,23 @@ For more information, see xref:../networking/ingress-controller-dnsmgt.adoc#ingr
 {product-title} 4.12 adds support for the following SR-IOV devices:
 
 * MT2892 Family [ConnectX&#8209;6{nbsp}Dx]
+* MT2894 Family [ConnectX-6 Lx]
 * MT42822 BlueField&#8209;2 in ConnectX&#8209;6 NIC mode
 * Silicom STS Family
 
 For more information, see xref:../networking/hardware_networks/about-sriov.adoc#supported-devices_about-sriov[Supported devices].
 
+[id="ocp-4-12-networking-supported-hardware-for-ovs"]
+==== Supported hardware for OvS (Open vSwitch) Hardware Offload 
+
+{product-title} 4.12 adds support for the following OvS Hardware Offload devices:
```

Comment:
I would say instead
"4.12 adds OvS Hardware Offload support for the following devices:"